.circleci/config.yml

```diff
@@ -44,6 +44,32 @@ jobs:
 
 
 
+  e2eTestCurPRDraining:
+    environment:
+      TESTED_VERSION: "current"
+      TEST_DIR: "integration/test/draining"
+    <<: *e2eTest
+
+  e2eTestCurMasterDraining:
+    environment:
+      TESTED_VERSION: "current"
+      TEST_DIR: "integration/test/draining"
+    <<: *e2eTest
+
+  e2eTestWIPPRDraining:
+    environment:
+      TESTED_VERSION: "wip"
+      TEST_DIR: "integration/test/draining"
+    <<: *e2eTest
+
+  e2eTestWIPMasterDraining:
+    environment:
+      TESTED_VERSION: "wip"
+      TEST_DIR: "integration/test/draining"
+    <<: *e2eTest
```

Comment:
This is the block for the draining tests.

Reply:
What's the difference between Master and PR?

Reply:
PR is run in PR. Master is run on merge.

Reply:
Yeah this is just how Circle's config works.